app/Controller/TagsController.php

```diff
@@ -894,7 +894,7 @@ private function __findObjectByUuid($object_uuid, &$type)
         return $object;
     }
 
-    public function attachTagToObject($uuid = false, $tag = false)
+    public function attachTagToObject($uuid = false, $tag = false, $local = 0)
```

Comment:
Why not to use `$local` variable as boolean?

Reply:
Indeed, I mapped what was in the database. I'll update tomorrow with Boolean values unless someone complain :-)